modules/sno-du-configuring-performance-addons.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-performance-addons_{context}"]
+= Configuring the Performance Addon Operator
+
+This is a key configuration for the single node distributed unit (DU).
+Many of the real time capabilities and service assurance is configured here.
+
+.Procedure
+
+* Configure the performance addons using the following example:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
```

Comment:
Does this CR exist or am I editing an existing CR?

Reply:
The user creates the file based on the example.